client/branded/src/global-styles/dropdown.scss

```
@@ -1,19 +1,10 @@
 @import 'bootstrap/scss/dropdown';
 
-.theme-light {
+:root {
     --dropdown-bg: var(--input-bg);
     --dropdown-border-color: var(--input-border-color);
     --dropdown-link-hover-bg: var(--color-bg-2);
     --dropdown-header-color: var(--text-muted);
```

Change to the `:root` selector here means that `--dropdown-bg` is initialized before `--input-bg` defined under `.theme-light`. And therefore, `--dropdown-bg` doesn't have any value. This leads to issues visible in the Chromatic [changeset](https://www.chromatic.com/pullrequest?appId=5f0f381c0e50750022dc6bf7&number=24271&view=changes). This can be fixed by moving CSS properties declaration under `.theme-*` selectors.

```suggestion
.theme-light,
.theme-dark {
    --dropdown-bg: var(--input-bg);
    --dropdown-border-color: var(--input-border-color);
    --dropdown-link-hover-bg: var(--color-bg-2);
    --dropdown-header-color: var(--text-muted);
```